src/parallel/ocbbs.cpp

```diff
@@ -890,12 +887,7 @@ static double broadcast(void*) {
 }
 
 static double checkpoint(void*) {
```

Comment:
@alkino : In the docs I see following two entries:

```
docs/hoc/modelspec/programmatic/network/parcon.rst:        Available only for the BlueGene.
docs/python/modelspec/programmatic/network/parcon.rst:        Available only for the BlueGene.
```

This `checkpoint()` function is just empty with return 0. I wonder if we should remove it from here and also from documentation rst file.

If we remove that, I think it has references in HOC layer:

```
src/oc/version.cpp: * hook for checkpoint
src/oc/hoc_init.cpp:    "checkpoint", hoc_checkpoint,
src/oc/hocdec.h:	int (*checkpoint)(void**);
src/oc/classreg.h:	int (*checkpoint)(void**),
src/oc/ocnoiv.cpp:void hoc_checkpoint(void) { hoc_ret(); hoc_pushx(0.);}
src/oc/ocnoiv.cpp:int hoc_readcheckpoint(char* f) { return 0; }
src/oc/ocfunc.h:extern void hoc_coredump_on_error(void), hoc_checkpoint(void), hoc_quit(void);
src/parallel/ocbbs.cpp:static double checkpoint(void*) {
src/parallel/ocbbs.cpp:	"checkpoint", checkpoint,
```

May be @nrnhines  or @alexsavulescu can help to remove related functions.

Reply:
I look but I will need help, yes

Reply:
@alexsavulescu helps me,